source/pmc/reporting.md

```diff
@@ -52,3 +52,50 @@ tool](https://whimsy.apache.org/board/agenda), or committed directly to
 the [subversion
 repository](https://svn.apache.org/repos/private/foundation/board).
 
+## What makes a good board report
+
+As with anything you write, the most important thing to keep in mind
+when writing your board report is who your audience is.
+
+The primary audience of your report is, of course, the board of
+directors. They are not, generally, experts on your particular technology. They are,
+rather, interested in the health and sustainability of the project.
+Questions to consider include:
+
+* Do you have enough active participants - both committers and PMC members -
+  to respond to urgent bugs or security?
+* Have you responded to any questions or comments that the board made on
+  your previous report?
+* Have you pointed out major achievements, or challenges, encountered in
+  the past quarter that an outsider would have missed?
+
+The secondary audience of your report is the general public. The minutes
+of board meetings are
+[published](https://apache.org/foundation/board/calendar.html) for our users,
+sponsors, the press, and the general public to read. Consider that
+audience when writing. For example, if there's a story that you'd like
+the press to tell about your project, you might mention it, and provide
+a quote that you'd like to see in an article.
+
+If there's something you wish to bring to the attention of the board,
+but isn't for public consumption, enclose it in &lt;private&gt; tags so
+that it won't be published in the public minutes.
+
+For example:
+
+```
+<private>
+Private tags should be alone on a line
+</private>
+```
+
+The report isn't just a once-a-quarter box to check -- it's your 
+opportunity to tell the story of your project,
+celebrate your accomplishments, ask for help or advice on challenges
+you're facing, and identify places where new contributors might find
+something to work on.
+
+We encourage you to look at [past board
+reports](https://apache.org/foundation/board/calendar.html) for inspiration from
+other projects' reports. 
```

Comment:
Great addition! Should we maybe link to https://whimsy.apache.org/board/minutes/ if people want to easily select another project and have a look at their past reports ?